reference/forms/types/collection.rst

```diff
@@ -286,6 +286,27 @@ the empty values will be kept.
     Read about the :ref:`form's empty_data option <reference-form-option-empty-data>`
     to learn why this is necessary.
 
+A value is deleted from the collection only if the normalized value is ``null``.
+However, you can also set the option value to a callable, which will be executed
+for each value in the submitted collection. If the callable returns ``false``,
```

Comment:
This looks wrong to me. The item is removed if the callable returns `true`, isn't?

Reply:
Yes, of course! Edited.